lib/order.rb

```diff
@@ -0,0 +1,53 @@
+require 'csv'
+
+require_relative 'customer'
+
+class Order
+  attr_reader :id, :products, :customer, :fulfillment_status
+
```

Comment:
I believe the project spec asks for `products`, `customer` and `fulfillment_status` to be writable as well as readable. Make sure you have a handle on the differences between `attr_reader` and `attr_accessor`.